src/pages/index.js

```diff
@@ -142,16 +156,26 @@ function getCardElement(data) {
   cardTitleEl.textContent = data.name;
 
   const cardLikeButtonEl = cardElement.querySelector(".card__like-button");
-  cardLikeButtonEl.addEventListener("click", () => {
-    cardLikeButtonEl.classList.toggle("card__like-button_active");
-  });
+  cardLikeButtonEl.addEventListener("click", (evt) =>
+    handleLike(evt, data._id)
+  );
 
+   if(data.isLiked) {
+    cardLikeButtonEl.classList.add("card__like-button_active");
+   }
 
   const cardDeleteButtonEl = cardElement.querySelector(".card__delete-button");
-  // cardDeleteButtonEl.addEventListener("click", (evt) => {
-  //   handleDeleteCard(cardElement, data._id);
-  //   cardElement.remove();
-  // });
+  cardDeleteButtonEl.addEventListener("click", (evt) => {
+    handleDeleteCard(cardElement, data._id);
+  });
+
+  deleteCancelBtn.addEventListener("click", function () {
+      closeModal(deleteModal);
+  })
+
+  deleteModalCloseBtn.addEventListener("click", function () {
+      closeModal(deleteModal);
+  })
```

Comment:
The `cancel` button and `close icon` of the **delete modal** should be handled on line 345 **outside** the `getCardElement()` function

Please, remove it from `getCardElement`

They should be handled only **once** in the file body, otherwise you add listeners again and again to the same elements. This may cause **`memory leak`**